provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -37,11 +38,68 @@ def _create_node_descriptions(self) -> List[NodeDesc]:
             nodes.append(self._create_node(node))
         return nodes
 
+    def _create_pool_descriptions(self) -> List[PoolDesc]:
+        pools: List[PoolDesc] = []
+        conf = self.provider
+        cluster_id = conf.get('cluster>cluster_id')
+        storage_set_count = int(
+            conf.get(f'cluster>{cluster_id}>site>storage_set_count'))
+
+        for x in range(storage_set_count):
+            storage_set_name = conf.get(
+                f'cluster>{cluster_id}>storage_set{x+1}>name')
+
+            data_devices_count: int = 0
+            for node in conf.get(
+                    f'cluster>{cluster_id}>storage_set{x+1}>server_nodes'):
+                for device in conf.get(
+                        f'cluster>{node}>storage>data_devices'):
+                    data_devices_count += 1
+
+            data_units_count = int(conf.get(
```

Comment:
[*suggestion*] just data_units and parity_units should be fine I think.

Reply:
I don't think that there is a specific naming rule in Python for these cases. But I also don't think that `nr` reads better than `count`. So I would suggest to keep it as is.